src/book/03-guides/03-hardware/02-flashing.mdx

```diff
@@ -163,3 +163,57 @@ The rest of these instructions assume a network using WPA2 (like the network use
      ```
    - Be sure to replace `<WiFi Interface>` with the name of the interface found earlier.
    - Wait a handful of seconds and run `ip addr` to check that the WiFi interface has an IP address
+
+
+## Switching Wifi Networks
+
+Switching wifi networks can be accomplished with the following two commands:
+
+```
+$ wpa_cli list_networks
+Selected interface 'wlp58s0'
+network id / ssid / bssid / flags
+0	epsilon-z	any	[CURRENT]
+1	epsilon-x	any	[DISABLED]
+2	NETGEAR89	any	[DISABLED]
+
+$ wpa_cli select_network 2
+Selected interface 'wlp58s0'
+client_loop: send disconnect: Broken pipe
+```
+
+The number `N` provided to `wpa_cli select_network <N>` is the network id given by `wpa_cli list_networks`. To switch to `epsilon-z` one would execute `wpa_cli select_network 0`.
+The example above was executed through a connection over the wifi interface, hence why the pipe broke.
+
```

Comment:
```suggestion

If the ip mask of the network is not `10.1.255.255` but instead `192.168.255.255` you need to change the static ip of the robot.
Edit `/etc/systemd/networkd/10-bond0.network` and change `10.1.X.X` to `192.168.X.X` keeping the `X` part the same. Now run `# systemctl restart networkd` to apply the changes.
When connecting back to the epsilon network these changes need to be reversed.

```
The ip mask may change, we should describe how to change the static ip.

Reply:
Hmm, that's interesting. I was trying to avoid modifying files so that rebooting the system would return it to the default state. I wonder if it's possible to obtain the subnet mask automatically? Maybe through DHCP? Idk.

Reply:
This is really only applicable for robocup now, and we will have to modify files anyway to add the robocup networks.
We don't need the spare router anymore, our main router now works for robot and game controller.